<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>PDFImportWidgetBase</class>
 <widget class="QWidget" name="PDFImportWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDFImportWidget</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Pages</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="boolAllPages">
        <property name="text">
         <string>All pa&amp;ges</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="boolFirstPage">
        <property name="text">
         <string>&amp;First page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="boolSelectionPage">
        <property name="text">
         <string>Selec&amp;tion of page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listPages">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Dimensions</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item column="0" row="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item column="0" row="1">
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>44</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item column="1" row="1">
           <widget class="KisIntParseSpinBox" name="intHeight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item column="0" row="0">
           <widget class="QLabel" name="intWidthzqffs">
            <property name="text">
             <string>Width:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item column="1" row="0">
           <widget class="KisIntParseSpinBox" name="intWidth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item column="0" row="1">
           <widget class="QLabel" name="intHeightqsdfq">
            <property name="text">
             <string>Height:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="KoAspectButton" name="pixelAspectRatioBtn" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" stdset="0">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>51</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item column="0" row="2">
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item column="1" row="0">
           <widget class="KisIntParseSpinBox" name="intResolution">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string>dpi</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item column="0" row="0">
           <widget class="QLabel" name="textLabel5">
            <property name="text">
             <string comment="horizontal resolution">Resolution:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item column="2" row="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>51</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header>KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>boolAllPages</tabstop>
  <tabstop>boolFirstPage</tabstop>
  <tabstop>boolSelectionPage</tabstop>
  <tabstop>listPages</tabstop>
  <tabstop>intResolution</tabstop>
  <tabstop>intWidth</tabstop>
  <tabstop>intHeight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>